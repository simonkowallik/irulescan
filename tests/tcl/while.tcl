while [expr 1] {
    expr 2
}
