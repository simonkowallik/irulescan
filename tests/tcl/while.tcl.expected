WARNING: Unquoted expr at `1` in `expr 1`
DANGEROUS: Dangerous unquoted expr at `[expr 1]` in `while [expr 1] {    expr 2}`
WARNING: Unquoted expr at `2` in `expr 2`

